src/core/config.js

```diff
@@ -64,7 +64,7 @@
 
     cdb.config = new Config();
     cdb.config.set({
-      cartodb_attributions: "CartoDB <a href='http://cartodb.com/attributions' target='_blank'>attribution</a>",
+      cartodb_attributions: "CartoDB <a href=\"http://cartodb.com/attributions\" target=\"_blank\">attribution</a>",
```

Comment:
Line is too long.<br>Identifier 'cartodb_attributions' is not in camel case.<br>Mixed double and single quotes.